airbyte-integrations/connectors/destination-mysql/src/test/java/io/airbyte/integrations/destination/mysql/MySQLDestinationTest.java

```diff
@@ -0,0 +1,81 @@
+package io.airbyte.integrations.destination.mysql;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.spy;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.google.common.collect.ImmutableMap;
+import io.airbyte.commons.json.Jsons;
+import java.util.Map.Entry;
+import org.junit.jupiter.api.Test;
+
+public class MySQLDestinationTest {
+
+  private MySQLDestination getDestination() {
+    final MySQLDestination result = spy(MySQLDestination.class);
+    return result;
+  }
+
+  private JsonNode buildConfigNoJDBCParameters() {
```

Comment:
Nitpick. The `JDBC` in the unit tests can be standardized as well.
```suggestion
  private JsonNode buildConfigNoJdbcParameters() {
````